vignettes/v1-2020-2021-mitma-data-codebook.qmd

```diff
@@ -0,0 +1,228 @@
+| `activity_destination` | `actividad_destino` | `factor` | The type of activity at the destination zone, similarly recoded as for `activity_origin` above. |
+| `residence_province_ine_code` | `residencia` | `factor` | The province code of residence if individuals who were making the trips in `n_trips`, encoded as province codes as classified by the Spanish Statistical Office (INE). |
+| `residence_province_name` | Derived from `residencia` | `factor` | The full name of the residence province, derived from the province code above. |
+| `time_slot` | `periodo` | `integer` | The time slot during which the trips occurred. |
+| `distance` | `distancia` | `factor` | The distance range of the trip, categorized into specific intervals such as `0005-002` (500 m to 2 km), `002-005` (2-5 km), `005-010` (5-10km), `010-050` (10-50 km), `050-100` (50-100 km), and `100+` (more than 100 km). |
+| `n_trips` | `viajes` | `numeric` | The number of trips for that specific origin-destination pair and time slot. |
+| `trips_total_length_km` | `viajes_km` | `numeric` | The total length of trips in kilometers, summing up all trips between the origin and destination zones. |
+| `year` | `year` | `integer` | The year of the recorded data, extracted from the date. |
+| `month` | `month` | `integer` | The month of the recorded data, extracted from the date. |
+| `day` | `day` | `integer` | The day of the recorded data, extracted from the date. |
+
+**Data transformation note**
+
+The original data is stored in the [`maestra-1`](https://www.transportes.gob.es/ministerio/proyectos-singulares/estudios-de-movilidad-con-big-data/estudios-de-movilidad-anteriores/covid-19/opendata-movilidad){target="_blank"} folder with suffixes `distritos` (for district zoning) and `municipios` (for municipality zoning). We only use the `district` level data because of several data issues with the `municipality` data documented [here](http://www.ekotov.pro/mitma-data-issues/issues/011-v1-tpp-mismatch-zone-ids-in-table-and-spatial-data.html){target="_blank"} and [here](http://www.ekotov.pro/mitma-data-issues/issues/012-v1-tpp-district-files-in-municipality-folders.html){target="_blank"}, but also because the distric level data contains more columns with useful origin-destination flow characteristics. As a result, you get both the `district` level data and the `municipality` level data with the same columns. `Municipality` level data is simply a re-aggregation of `district` level data using the official relations file where `district` identifiers are mapped to `municipality` identifiers (orginal file is [`relaciones_distrito_mitma.csv`](https://opendata-movilidad.mitma.es/relaciones_distrito_mitma.csv){target="_blank"}).
+
+**Getting the data**
+
+To access the data, use the `spod_get_od()` function. In this example we will use a short interval of dates:
```

Comment:
spod_get_od() or spod_get()? According to the reference vignette spot_get() is to retrieves the origin-destination data and spod_get_od() to load the OD v1 data for specific dates. This is confusing. 

Also, spod_get_od() is not in Figure 1.

Reply:
Good catch of `spod_get_od()`. This was just a typo. I removed `spod_get_od()` in favour of having just one single function `spod_get()` which has `type` argument that you can set to "od" or other data available in the data set. So here it is just a mistake.

However, regarding the naming. Perhaps it is good to have `spod_get_od()` that simply runs `spod_get(type='od'). That would be easier to type and better for autocomplete... Or perhaps that would be more confusing, I'm not sure.

Reply:
Not sure either. If you use spot_get_od() you will have to find a spot_get_[something_else]() for the non-od data.